packages/client/src/views/Home/RecordAudit.tsx

```diff
@@ -455,84 +402,162 @@ const getApplicationInfo = (
   )
 }
 
-export const ShowRecordAudit = (props: IFullProps) => {
-  const applicationId = props.match.params.applicationId
-  let application: IApplicationData | null
-  application = getSavedApplications(props)
-  const isDownloaded = application ? true : false
-  if (!isDownloaded) {
-    application = getWQApplication(props)
-  }
-
+function RecordAuditBody({
+  clearCorrectionChange,
+  declaration,
+  isDownloaded = false,
+  intl,
+  goToCertificateCorrection
+}: {
+  declaration: IDeclarationData
+  isDownloaded?: boolean
+  intl: IntlShape
+} & IDispatchProps) {
   const actions = []
   if (
     isDownloaded &&
-    application &&
-    application.status === SUBMISSION_STATUS.REGISTERED
+    (declaration.status === SUBMISSION_STATUS.REGISTERED ||
+      declaration.status === SUBMISSION_STATUS.CERTIFIED)
   ) {
     actions.push(
       <TertiaryButton
         id="btn-correct-record"
         align={ICON_ALIGNMENT.LEFT}
         icon={() => <Edit />}
         onClick={() => {
-          props.clearCorrectionChange(applicationId)
-          props.goToCertificateCorrection(
-            applicationId,
-            CorrectionSection.Corrector
-          )
+          clearCorrectionChange(declaration.id)
+          goToCertificateCorrection(declaration.id, CorrectionSection.Corrector)
         }}
       >
-        {props.intl.formatMessage(correctionMessages.title)}
+        {intl.formatMessage(correctionMessages.title)}
       </TertiaryButton>
     )
   }
   return (
-    <div id={'recordAudit'}>
+    <Content
+      title={declaration.name || 'No name provided'}
+      titleColor={declaration.name ? 'copy' : 'grey600'}
+      size={ContentSize.LARGE}
+      topActionButtons={actions}
+      icon={() => (
+        <ApplicationIcon
+          color={STATUSTOCOLOR[(declaration && declaration.status) || 'DRAFT']}
+        />
+      )}
+    >
+      {getDeclarationInfo(declaration, isDownloaded, intl)}
+    </Content>
+  )
+}
+
+function getBodyContent({
+  clearCorrectionChange,
+  declarationId,
+  draft,
+  goToCertificateCorrection,
+  language,
+  tab,
+  intl,
+  resources,
+  workqueueDeclaration
+}: IFullProps) {
+  const actionProps = {
+    clearCorrectionChange,
+    goToCertificateCorrection
+  }
+  if (!draft && tab === 'search') {
+    return (
+      <>
+        <Query
+          query={FETCH_DECLARATION_SHORT_INFO}
+          variables={{
+            id: declarationId
+          }}
+          fetchPolicy="no-cache"
+        >
+          {({
+            loading,
+            error,
+            data
+          }: {
+            loading: any
+            error?: any
+            data: any
+          }) => {
```

Comment:
`{({
            loading,
            error,
            data
          }) => {`

I think here we do not have to define a type to any!